src/main/java/com/xwiki/authentication/saml/samlauth/Saml2XwikiAttributes.java

```diff
@@ -0,0 +1,16 @@
+package com.xwiki.authentication.saml.samlauth;
+
+import java.util.Map;
+import java.util.Set;
+
+public class Saml2XwikiAttributes {
```

Comment:
To keep it more consistent with the rest of the code:
```suggestion
public class Saml20XWikiAttributes {
```